src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -586,6 +685,11 @@ private void showConnectionLostErrorMessage() {
                                              Localization.lang("Connection to OpenOffice/LibreOffice has been lost. " + "Please make sure OpenOffice/LibreOffice is running, and try to reconnect."));
     }
 
+    private void showNoDocumentErrorMessage() {
+        dialogService.showErrorDialogAndWait(Localization.lang("Connection to document lost"),
+                                             Localization.lang("Connection to OpenOffice/LibreOffice document has been lost. " + "Please make sure OpenOffice/LibreOffice is running, and try to reconnect."));
```

Comment:
An extra `+` between the Strings

Reply:
Yes, copied from the message above. May have been two lines
before the long-lines group took over. Or may have been to messages to localize.

Removed both.